src/urllib3/util/retry.py

```diff
@@ -143,7 +143,7 @@ class Retry:
         (most errors are resolved immediately by a second try without a
         delay). urllib3 will sleep for::
 
-            {backoff factor} * (2 ** ({number of total retries} - 1))
+            {backoff factor} * (2 ** ({number of previous retries}))
```

Comment:
Between the first and second attempts, there will be no previous retries since the second attempt will be the first retry 🙂

What do you think about using "number of previous attempts" in the formula like an AWS blog post [here](https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/) does?

Reply:
Thanks for following up @illia-v! Right - between the first and second attempts there will be no previous retries. Is this a problem for my proposed change? The docs above the line that I've changed [specify](https://github.com/urllib3/urllib3/blob/ac675f834ff59f9526b183a8b933b9be840f2d32/src/urllib3/util/retry.py#L142) that this formula only applies to the times *after* the second attempt so I don't think this is a problem (but happy to change it if it is unclear).

That being said - I think your suggestion of using "number of previous attempts" is more clear, so I'll update this PR to use that. Thank you!

Reply:
No problem - thanks for reviewing!